rest/views.py

```diff
@@ -819,7 +838,17 @@ def make_facet_counts(self, queryset):
             queryset = queryset.facet(vocabulary.id)
 
         facet_counts = queryset.facet_counts()
-        ret = get_vocabularies(facet_counts)
+        vocabs = get_vocabularies(facet_counts)
+
+        # return dictionary
+        ret_dict = {}
+
+        # process vocabularies
+        for key, values in vocabs.items():
+            missing_count = queryset.filter(
```

Comment:
Right now, it has to do a query per vocabulary. I don't know what the performance effect is (have you timed it, @giocalitri?). In any case, this needs to be in before the Elasticsearch-DSL work is done, so it's only temporary.